conf/messages.en

```diff
@@ -153,9 +153,9 @@ howManyEntries.checkYourAnswersLabel = Number of entries
 howManyEntries.error.required = Enter the number of entries you are claiming for
 howManyEntries.error.length = Number of entries must be between 2 and 999,999
 
-entryDetails.single.title = Entry details
+entryDetails.single.title = Add the details for this entry
 entryDetails.multiple.title = Enter the details of the oldest entry
```

Comment:
Did we check if the multiple entry title/heading needed changing?  e.g. "Add the details of the oldest entry"

Reply:
No. I believe this is RAD feedback. I can see that at the RAD they might say that simply "Entry details" is insufficient. I doubt they would balk at "Enter the details of the oldest entry"

I just treated it as a simple content change, rather than a bring consistancy ticket. There is a lot of inconsistancy if you go looking for it.

So to end with the answer "No"